VIDEO_2_SCRIPT.md

```diff
@@ -0,0 +1,143 @@
+
+- **Deployment**: Due to the low coupling between components, deployment is possible without worrying about dependencies or preconditions.
+
+- **Flexibility**: EDA allows rapid response to a changing environment, because each event processing component has a single responsibility and is completely decoupled from the others.
+
+The most significant disadvantages associated with this model are the difficult testability of the tests. Being an asynchronous model adds more complexity to this part. The lack of guarantee of obtaining a response to the sender and the scarcity of recovery support in case of partial failure is the most significant drawback.
+
+Arguably its biggest handicap is the inability to predict when a request or reaction to an event will arrive; these could occur at any time. For this reason, it is often necessary to implement a queuing system at times of heavy traffic.
+
+## ASYNCAPI and EDA
+
+AsyncAPI is an open-source initiative that seeks to improve the current state of Event-Driven Architectures (EDA). Its long-term goal is to make working with EDAs as easy as it is to work with REST APIs. That goes from documentation to code generation and from discovery to event management. Most of the processes you apply to your REST APIs nowadays would be applicable to your event-driven asynchronous APIs too.
+
+To make this happen, the first step has been to create a specification that allows developers, architects, and product managers to define the interfaces of the EDA interfaces using AsyncAPI. Much like OpenAPI (fka Swagger) does for REST APIs. 
+
+The specification settles the base for a greater and better tooling ecosystem for EDA's. 
+
+The event architecture allows you to create more accessible applications with less coupling between services, thus enabling a highly flexible architecture. These fundamental characteristics make this model the best choice for AsyncAPI development.
+
+One of the reasons for the success of REST APIs has been the existence of standards like OpenAPI that allow you to edit specifications, create documentation automatically, mock generators, etc. And it is the contribution to this success that is the main driver of AsyncAPI. Currently, the strategy is to continue improving, implementing resources that make it more complete and solid. 
+
+## Goodbye to the viewer
+
+You might be looking for a solution to automate and formalize the documentation or code generation of your event-driven (micro)services. Likewise, you might be aiming to establish solid standards for your events and improve the governance of your asynchronous APIs. If so, AsyncAPI is here to help you.
+
+Vice versa, we are looking forward to your feedback and contributions. If you have not yet joined the AsyncAPI community through its Slack or GitHub Discussions, we invite you to participate in the growth of the specification.
+
+See you in the next video!
```

Comment:
We hope you find these videos useful and we look forward to seeing you in the next one!